reconcile/test/aws_saml_idp/test_aws_saml_idp_integration.py

```diff
@@ -0,0 +1,66 @@
+from collections.abc import Callable
+
+import httpretty as _httpretty
+
+from reconcile.aws_saml_idp.integration import AwsSamlIdpIntegration, SamlIdpConfig
+from reconcile.gql_definitions.aws_saml_idp.aws_accounts import AWSAccountV1
+
+
+def test_aws_saml_idp_get_early_exit_desired_state(
+    intg: AwsSamlIdpIntegration, fixture_query_func: Callable
+) -> None:
+    state = intg.get_early_exit_desired_state(query_func=fixture_query_func)
+    assert "accounts" in state
+
+
+def test_aws_saml_idp_get_aws_accounts(
+    gql_class_factory: Callable, aws_accounts: list[AWSAccountV1]
+) -> None:
+    assert aws_accounts == [
```

Comment:
Only assert in test is a little confusing, a clear test structure is `Arrange-Act-Assert`, arrange can be done via fixtures, but if `Act` is also hidden in fixture, it's hard to tell what case is being tested here.

Reply:
Fixed in [707c9d5](https://github.com/app-sre/qontract-reconcile/pull/4060/commits/707c9d537869af8e9c7cb6adb734a36a0dff6a34)